### DISCUSSION TITLE
Easier Changes

### DISCUSSION THREAD

**Discussion by aarushbhutra** [community/community#11045] [Copilot Conversations] [answered]
I think it will be useful if after a set amount of time like 20 seconds if the user doesn't accept it should not show it and in intellij there isnt really a easy way to hide a suggestion or get a different one.

**Comment by jansorg** [comment, marked as answer]
@aarushbhutra Thanks! You can press Escape or change the content to hide a suggestion. There are a few more shortcuts at `Preferences > Keymap > Plugins > GitHub Copilot`. Are the shortcut good enough for your use-case or would you still prefer something else?
